packages/manager/modules/bm-server-components/src/ipmi/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,71 @@
+{
+  "server_configuration_impi_loading_error": "Une erreur est survenue lors de la récupération des informations",
+  "server_configuration_impi_reset_interfaces": "Redémarrer IPMI",
+  "server_configuration_impi_reset_sessions": "Supprimer les sessions",
+  "server_configuration_impi_off": "Le module IPMI n'est pas disponible pour ce serveur.",
+  "server_configuration_impi_off_kvm": "Toutefois, si vous souhaitez bénéficier de cette fonctionnalité, vous pouvez utiliser un KVM.",
+  "server_configuration_impi_order_kvm": "Commander un KVM",
+  "server_configuration_impi_disabled": "Le statut du serveur est <b>{{t0}}</b>. Le module IPMI est donc désactivé.",
+  "server_configuration_impi_test": "Tester IPMI",
+  "server_configuration_impi_http": "Test d'accès au module IPMI (http)",
+  "server_configuration_impi_password": "Initialisation d'une session IPMI (authentification)",
+  "server_configuration_impi_ping": "Test d'accès au module IPMI (ping)",
+  "server_configuration_impi_information": "IPMI permet de se connecter à votre serveur via les méthodes suivantes :",
+  "server_configuration_impi_information_java": "IPMI permet de se connecter à votre serveur via l'applet JAVA suivant :",
+  "server_configuration_impi_information2": "En initialisant une session 'Remote KVM' (accès au clavier, à la souris et à l'affichage de votre serveur à distance) depuis votre IP {{t0}}<ul><li>Depuis une applet Java Web Start</li></ul>",
+  "server_configuration_impi_information3": "En initialisant une session 'Serial Over LAN' (accès à la console série de votre machine via une connexion sécurisée) depuis votre IP {{t0}}<ul><li>Depuis votre navigateur (https)</li></ul>",
+  "server_configuration_impi_start": "Démarrer",
+  "server_configuration_impi_navigation": "Depuis votre navigateur (SoL)",
```

Comment:
SoL here and we have SOL (L. 21).
We have to chose one for consistency, may be SOL.

Reply:
Good suggestion. This is legacy code. I have just moved/split it to new file and module. The aim of the refactoring is to do with minimal code changes. This can be taken as a separate task.